src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageClient.java

```diff
@@ -55,6 +55,9 @@
   @JsonRequest("sonarlint/showTaintVulnerability")
   CompletableFuture<Void> showTaintVulnerability(ShowAllLocationsCommand.Param params);
 
+  @JsonRequest("sonarlint/notIgnoredByScm")
+  CompletableFuture<Boolean> notIgnoredByScm(String fileUri);
```

Comment:
The name of the method looks strange to me. Why not inverting the name to `isIgnoredByScm`?